### PR TITLE
Announce the agenda of 2020 Spark+AI summit VIRTUAL

### PR DESCRIPTION
**Pull request by gatorsmile**
This PR is to add the agenda info of Spark+AI Summit. 

image

### CODE REVIEW

**Comment by gatorsmile** [comment]
cc @yhuai @cloud-fan @srowen

**Review by srowen — APPROVED** [review]

File: news/_posts/2020-06-15-spark-ai-summit-june-2020-agenda-posted.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: Spark+AI Summit (June 22-25th, 2020, San Francisco) agenda posted
```
Comment (holdenk) [inline news/_posts/2020-06-15-spark-ai-summit-june-2020-agenda-posted.md]: It's not really in San Francisco is it?

Comment (srowen) [reply news/_posts/2020-06-15-spark-ai-summit-june-2020-agenda-posted.md]: Hehe good point. It's online / virtual.

Comment (holdenk) [reply news/_posts/2020-06-15-spark-ai-summit-june-2020-agenda-posted.md]: Yeah, I think calling that out would be good, so we can have more of the community participate. People may be used to skipping events outside of their local and it's nice that we can have a more inclusive event this time :)

**Review by maropu — APPROVED** [review]
Oh, will be held soon, nice. LGTM except for the existing comment.

File: news/_posts/2020-06-15-spark-ai-summit-june-2020-agenda-posted.md (hunk 1 above)
Comment (cloud-fan) [reply news/_posts/2020-06-15-spark-ai-summit-june-2020-agenda-posted.md]: yea it's virtual, but AFAIK the time schedule of the talks still follows the SF timezone. Shall we mention that as well?

Comment (HyukjinKwon) [reply news/_posts/2020-06-15-spark-ai-summit-june-2020-agenda-posted.md]: But looks like the timezone isn't mentioned in the summit site either. I guess we can just simply follow what the site says.

**Review by HyukjinKwon — APPROVED** [review]
LGTM except that comment.

File: news/_posts/2020-06-15-spark-ai-summit-june-2020-agenda-posted.md (hunk 1 above)
Comment (gatorsmile) [reply news/_posts/2020-06-15-spark-ai-summit-june-2020-agenda-posted.md]: fixed

**Review by srowen — APPROVED** [review]

**Pull request merged**